go.mod

```diff
@@ -3,8 +3,7 @@ module github.com/IBM/secret-common-lib
 go 1.16
 
 require (
-	github.com/IBM/secret-utils-lib v0.0.0-20220411122140-7d2c0d0e5d07
-	github.com/pierrre/gotestcover v0.0.0-20160517101806-924dca7d15f0 // indirect
+	github.com/IBM/secret-utils-lib v0.0.0-20220506111001-93b93a0e312f
```

Comment:
can you create the release for secret-utils-lib repo and update the dependency here

Reply:
Updated the library.